arch/platform/sky/Makefile.common

```diff
@@ -162,11 +162,8 @@ else
 	@echo "Use the \"ipconfig\" command to find out the IP addresses of the local interfaces"
 endif
 
-linslip: $(CONTIKI)/tools/tunslip
-	$(CONTIKI)/tools/tunslip -s $(USBDEVPREFIX)$(firstword $(CMOTES)) 172.16.0.0 255.255.0.0
-
-$(CONTIKI)/tools/tunslip:
-	(cd $(CONTIKI)/tools; $(MAKE) tunslip)
+linslip: $(TUNSLIP6)
+	$(TUNSLIP6) -s $(USBDEVPREFIX)$(firstword $(CMOTES)) 172.16.0.0 255.255.0.0
```

Comment:
Are you sure about this? I seem to remember a separate tool called `tunslip` (as opposed to `tunslip6`)